scripts/cloud-release.sh

```diff
@@ -20,3 +21,13 @@ function triggerTagRelease {
   fi
   triggerProjectBatch $RELEASE_ACCOUNT_PROD $RELEASE_ROLE_NAME "${RELEASE_PROFILE_NAME}Prod" $RELEASE_PROJECT_NAME $branch_name
 }
+
+function deprecateRelease {
+  DEPRECATION_MESSAGE=$1
+  SEARCH_FOR_RELEASE_STARTING_FROM=$2
+  USE_NPM_REGISTRY=$3
```

Comment:
Are these the arguments to `yarn trigger-deprecate-release` script?

Reply:
Yes